libs/ui/widgets/src/lib/ui-widgets.module.ts

```diff
@@ -41,7 +39,6 @@ import { CommonModule } from '@angular/common'
     StepBarComponent,
     LoadingMaskComponent,
     PopupAlertComponent,
-    BadgeComponent,
```

Comment:
Wondering about this, this is making a switch to standalone components mandatory, while still exporting from modules could allow both use.

Reply:
The way I understand it, standalone components when used by non-standalone ones can either be part of an "outgoing" module (here UI widgets) or an "incoming" module (for instance an app module).

I feel like a benefit of converting things to standalone is that the lib modules will become slimmer and eventually disappear. For that to happen we need to import the standalone components as close as possible to their consumer, which would mean in the app module.

I hope I'm clear enough :grimacing:

Reply:
OK, if its a conscious choice to get definitely rid of Angular modules, this is the way to go indeed.